egs2/TEMPLATE/asr1/db.sh

```diff
@@ -43,6 +43,7 @@ LIBRILIGHT_LIMITED=
 FSC=
 SLURP=
 VOXCELEB=
+MINDS14_DIR=${PWD}/MInDS-14
```

Comment:
I don't think this PWD would be functioning

Reply:
@ftshijt Hi. I just fixed the path and confirmed that it functions as intended.